### PR TITLE
Add missing validation documentation

### PR DESCRIPTION
**Pull request by sergei-maertens**
Most of these validations are currently implemented, but missing from the documentation right now. Some of the validations need to be added, so I think this will be 1.0.1 for the relevant APIs.

### CODE REVIEW

File: docs/_content/standaard/besluiten/index.md [hunk 1]
```diff
@@ -142,9 +142,35 @@ Merk op dat het aanmaken van de relatie niet gelimiteerd is tot het aanmaken
...
+`Besluit.besluittype` voorkomen in de `Besluit.zaak.zaaktype.besluittypen`.
+
+
+**<a name="brc-007">Valideren dat het `informatieobjecttype` van een `BesluitInformatieObject` bij het `Besluit.besluittype` hoort ([brc-007](#brc-007))</a>**
+
+<span style="padding: 0.2em 0.5em; border: solid 1px #FF6600; border-radius: 3px; background: #FFFF99;">
+    <strong>Documentatie toegevoegd in patch 1.0.1</strong>
+</span>
+
+Wanneer een `BesluitInformatieObject` toevoegd wordt, dan MOET het
```
Comment (stevenbal) [inline docs/_content/standaard/besluiten/index.md]:
```suggestion
Wanneer een `BesluitInformatieObject` toegevoegd wordt, dan MOET het
```

**Comment by joeribekker** [comment]
@stevenbal Komen deze wijzigingen (in nummers en tekstueel) verder al wel overeen met de testscenario's?

**Comment by stevenbal** [comment]
@joeribekker nee nog niet, zal ik vandaag aan werken

**Comment by sergei-maertens** [comment]
@stevenbal typo fixed - do you have a status update on the postman tests?

**Comment by stevenbal** [comment]
@sergei-maertens I've renamed the tests where needed and added new tests for the validation in this PR. For OZ everything except zrc-018 (validation when adding zaakeigenschap) passes

**Comment by sergei-maertens** [comment]
Sweet, lgtm!

**Pull request merged**